src/index.js

```diff
@@ -72,15 +72,34 @@ const getLooseMode = looseOpts => {
   return looseOpts;
 }
 
-// TODO: Allow specifying modules as: Boolean, String or Array of module types
-const getModules = modulesOpts => {
+/**
+ * Given modulesOpts return an Array of all required module transformations
+ * @param  {String|Array.<String>|Boolean}  modulesOpts  An argument enumerating the modules to transform
+ * @return {Array.<String>}  An array of module transformation plugin names
+ */
+export const getModuleTransformations = modulesOpts => {
   if (!modulesOpts) { return []; }
-  return modulesOpts;
+
+  if (modulesOpts === true) { return Object.values(MODULE_TRANSFORMATIONS); }
```

Comment:
No problem - I was away for the long weekend! I DID misunderstand this. I understood it to be UMD/CJS/etc => cjs and not ES2015 => UMD/CJS/etc. 

Now I know. Hope you had a great labor day!